gallery/gallery/lib/pages/demo.dart

```diff
@@ -506,49 +482,52 @@ class _DemoSectionOptions extends StatelessWidget {
     final textTheme = Theme.of(context).textTheme;
     final colorScheme = Theme.of(context).colorScheme;
 
-    return Container(
-      constraints: BoxConstraints(maxHeight: maxHeight),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          Padding(
-            padding: const EdgeInsetsDirectional.only(
-              start: 24,
-              top: 12,
-              end: 24,
-            ),
-            child: Text(
-              GalleryLocalizations.of(context).demoOptionsTooltip,
-              style: textTheme.display1.apply(
-                color: colorScheme.onSurface,
-                fontSizeDelta:
-                    isDisplayDesktop(context) ? desktopDisplay1FontDelta : 0,
+    return Align(
```

Comment:
What is the `Align` for?

Reply:
A Container within an Expanded will fill the space, even if you set width/height constraints, unless you use an Align.